local_setup.md

```diff
@@ -0,0 +1,193 @@
+    </CORSRule>
+</CORSConfiguration>
+```
+1.  Now click save, then close and after that the blue save button.
+
+### Set UP Amazon IAM
+
+1.  Log in to amazon if you aren't already because you just set up your s3 bucket.
+
+1.  Navigate to IAM by clicking on the "Identity & Access Management" icon:
+
+  ![IAM-icon](IAM-icon.png)
+
+1.  Navigate to Users in the navigation on the left side.
+
+  ![Users in Navigation](users-in navigation.png)
+
+1.  Click on the blue button with a white "create new users"-writing:
+
+  ![Create New Users](better-create-new-users-button.png)
+
+1.  Enter your name into the first field.
+
+  ![Insert Your Name](insert-your-name.png)
+
+1.  Press "create" which you'll find in the lower right corner. It's blue with white writing:
+
+  ![Create Button](better-create-button.png)
+
+1.  Click on "Show User Security Credentials" and you see your needed keys. **IMPORTANT: **Download and store them in a secure place.
+
+1.  Now click on your newly created IAM User and navigate to "Permissions". There you open the "Inline Policies" by clicking on the arrow beside it (if its not already opened)
+
+  ![Inline Policies](more-around-inline-policies.png)
+1.  Now click on the "click here" link in the "Inline Policies" section. 
+
+1.  Choose "Custom Policy" and click select.
+
+  ![Select Custom Policy](select-custom-policy.png)
+
+1.  Set the name of your Policy to something like: ```<yourname>-s3-policy```. 
+
+1.  Insert this text to "Policy Document:
+```json
+{
+  "Statement": [
+    {
+      "Action": [
+        "s3:ListAllMyBuckets"
+      ],
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::*"
+    },
+    {
+      "Action": "s3:*",
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::<your-s3-bucket-name>"
+    },
+    {
+      "Action": "s3:*",
+      "Effect": "Allow",
+      "Resource": "arn:aws:s3:::<your-s3-bucket-name>/*"
+    }
+  ]
+}
+```
+It should look like this:
+
+  ![New Policy](insert-new-policy.png)
+
+1.  Click on "Apply Policy"
+
+### Configure renuo-upload-signing
+
+Now you have to use the newly created bucket in your renuo-upload-signing by setting following keys in the *.env*-File:
+
+-  **S3_BUCKET_NAME:** needs the name you gave to your bucket.
+-  **CDN_HOST** differs if you changed the bucket-location for example. The part after the ```/``` will always be your bucket-name though.
+-  **S3_PUBLIC_KEY** needs the "Access Key ID" which you downloaded before.
+-  **S3_SECRET_KEY** needs the "Secret Access Key" which you downloaded before.
+
+The *.env* file exists after running `bin/setup` as a copy of *example.env*. Adjust it:
+
+```rb
+S3_BUCKET_NAME: 'renuo-upload-<yourname>-development'
+S3_PUBLIC_KEY: 'your public key found in Amazon IAM'
+S3_SECRET_KEY: 'your secret key found in Amazon IAM'
+CDN_HOST: 's3.eu-central-1.amazonaws.com/renuo-upload-<yourname>-development' #without https://, just the domain
+```
+
+### Set Up renuo-cms-demo as Example Project
```

Comment:
this is optional, right? if yes, mark it this way.

also: this should be documented in the renuo-cms-demo